gamestonk_terminal/jupyter/jupyter_controller.py

```diff
@@ -0,0 +1,240 @@
+"""Jupyter Controller Module"""
+__docformat__ = "numpy"
+
+import argparse
+import difflib
+from typing import List, Union
+
+from prompt_toolkit.completion import NestedCompleter
+
+from gamestonk_terminal import feature_flags as gtff
+from gamestonk_terminal.helper_funcs import (
+    get_flair,
+    system_clear,
+)
+from gamestonk_terminal.menu import session
+
+# pylint: disable=too-many-public-methods,import-outside-toplevel
+
+
+class JupyterController:
+    """Resources Controller class"""
+
+    # Command choices
+    CHOICES = [
+        "cls",
+        "home",
+        "h",
+        "?",
+        "help",
+        "q",
+        "quit",
+        "..",
+        "exit",
+        "r",
+        "reset",
+    ]
+
+    CHOICES_COMMANDS = [
+        "reports",
+        "dashboards",
+        "finra",
+        "edgar",
+        "fred",
+        "learn",
+        "econiverse",
+    ]
+
+    CHOICES += CHOICES_COMMANDS
+
+    def __init__(self, queue: List[str] = None):
+        """Constructor"""
+        self.rc_parser = argparse.ArgumentParser(add_help=False, prog="jupyter")
+        self.rc_parser.add_argument(
+            "cmd",
+            choices=self.CHOICES,
+        )
+        self.queue = queue if queue else list()
+        self.completer: Union[None, NestedCompleter] = None
+
+        if session and gtff.USE_PROMPT_TOOLKIT:
+            choices: dict = {c: {} for c in self.CHOICES}
+
+            self.completer = NestedCompleter.from_nested_dict(choices)
+
+    @staticmethod
+    def print_help():
+        """Print help"""
+        help_str = """
+Jupyter:
+   reports       creates jupyter reports
```

Comment:
This spacing is not the correct one and you are missing the ">" which distinguishes menu from command:
```
>   resources
    
(✨) / $ jupyter

Jupyter:
   reports       creates jupyter reports
   dashboards    shows interactive jupyter dashboards  
```

Should be:
```
>   reports       creates jupyter reports
>   dashboards    shows interactive jupyter dashboards  
```